public/login.php

```diff
@@ -17,8 +17,19 @@
                              array($input["email"]));
       if (array_key_exists("email", $data) && array_key_exists("wachtwoord", $data)){
         if (verifyPassword($input["password"], $data["wachtwoord"]) && $data["email"] == $input["email"]){
-            // -- start setting session from this point -- //
-            $message = "LOGGEDIN";
+            $data = getTableRecord("SELECT id, ingelogd FROM gebruiker WHERE email = ?", "s", array($input["email"]));
+            // set $_SESSION['id'] for getting the correct details of a user
+            $_SESSION['id'] = $data["id"];
+            if ($data["ingelogd"] == 1) {
```

Comment:
Moet die niet een error geven als die al is ingelogd, omdat die dan waarschijnlijk op een andere apparaat is ingelogd